src/microbe_stage/editor/PatchMapNode.cs

```diff
@@ -75,6 +91,16 @@ public bool Marked
         }
     }
 
+    public bool SelectionAdjacent
```

Comment:
"SelectionAdjacent" doesn't tell me what this is. This should be probably renamed and explanation comment added. Is this a bool for when the patch is adjacent to the selected patch? If so this should be named "AdjacentToSelectedPatch"
also the related variables should be updated.